DataDog/integrations-core
# florentclarret/weaviate/update prs

- generate boilerplate
- working ddev
- add tests
- use py3.9
- add API metadata
- POC weaviate
- POC weaviate
- metadata entries
- add service_checks.json
- WIP
- metadata entries
- touch ups and configs
- fix config
- require metrics endpoint
- fix tests
- refactor
- nits
- add validation and param name change
- tests
- whitespace
- add auth env
- styling
- metric rename
- validate dashboards
- dashboards placeholder
- test
- styling
- tests
- extra time
- shorten time
- test
- revert
- further reverts
- formatting
- change response check from 200 to ok
- styling
- re-order api calls
- styling
- test for liveness
- add a sleep
- small refactor
- small changes
- add undocumented metrics
- fix tests with new metrics
- cleanup
- Update weaviate/tests/utils.py
- nits
- rename test
- nits
- styling
- nits
- raise on error
- revert raise
- expand fixtures
- get validations passing
- fix link
- refactor module
- nits
- get test passing
- change fixture file extensions
- delete file
- fix ci
- fix license headers
- change tags from api to line up with OMv2
- Address comments

### What does this PR do?


### Motivation


### Additional Notes


### Review checklist (to be filled by reviewers)

- [ ] If the PR doesn't need to be tested during QA, please add a `qa/skip-qa` label.
